FAQ.md

```diff
@@ -1,37 +1,70 @@
+The Code of Conduct Committee is responsible for maintaining the Code of Conduct as well as deciding if a given use of ml5.js violates the Code of Conduct. It can make those decisions in response to a specific request, or through an investigation it initiates directly.
+
+The Code of Conduct Committee’s members are:
+
+- Christina Dacanay
+- Lydia Jessup
+- Sam Krystal
+- Joey Lee
+- Ashley Jane Lewis
+- Bomani Oseni McClendon
+- Daniel Shiffman
+- Michael Weinberg
+
+If you believe that a project that incorporates ml5.js violates the Code of Conduct, or are concerned that your own project may violate the Code of Conduct, we ask that you report it by emailing info@ml5js.org. Please include your name and a description of the incident.
+
+## How is ml5.js funded?
+
+The development of the ml5.js project has been made possible through financial and “in kind” support from institutions.
+
+### ITP/IMA at NYU
+
+The project began as a series of short experiments by students and faculty at ITP/MA which combined elements of TensorFlow.js (then called ‘deeplearn.js’) and p5.js in examples for the coding curriculum. Since then development of the project has almost entirely happened through for-credit internships, student work-study, class assignments, staff administrative support, and informal volunteer clubs and workshops at ITP/IMA itself.
+
+### Google
+
+The ml5.js codebase itself is almost entirely built on top of TensorFlow.js, an open source machine learning library in JavaScript maintained by researchers and employees of Google (Alphabet, Inc.). TensorFlow.js, itself, is a subset of the larger TensorFlow machine learning ecosystem across many programming languages and frameworks, also maintained by Google. Funding from the form of two unrestricted cash gifts from Google University Relations has paid for the bulk of expenses associated with the support from ITP/IMA. This funding was made possible through our collaboration with the TensorFlow.js research and development team which is part of People + AI Research (PAIR) at Google.
+
+### Processing Foundation
+
+The ml5.js library, though now broadly applicable in native JavaScript, was originally intended as a “plug-in” to add machine learning capabilities to the general purpose creative coding library p5.js, maintained and developed by The Processing Foundation. The Processing Foundation has contributed to the development of ml5.js in a myriad of ways via the intersection of the ml5.js and creative coding communities. Processing Foundation fellowships and other community events have featured and provided support for ml5.js.
+
+### COSA
+
+The ml5.js project has received support from the [Clinic for Open-Source Arts](https://www.du.edu/ahss/opensourcearts/).
```

Comment:
Addition:
* we should also include: 

```
### Frank-Ratchye STUDIO for Creative Inquiry

The ml5.js project has received support from the [Frank-Ratchye STUDIO for Creative Inquiry](https://studioforcreativeinquiry.org/)

```

Reply:
Added with https://github.com/ml5js/Code-of-Conduct/pull/28/commits/640a654345bf1b6a6708bbcabb84ed67a9b44476